### ISSUE TITLE
[BUG] bunkerweb.service reported that there is missing configuration file

### ISSUE THREAD

**Issue by snow6326** [bunkerity/bunkerweb#954]
### What happened?

Hello all, I am a newcomer to Bunkerweb's user community. Yesterday, I completed the installation and enabled the web UI based on the documentation page. I successfully logged into the Web UI and created a new web instance. I attempted to proxy the traffic from an existing web application with SSL protection (the original web application already has an SSL certificate). I tried to pass the traffic through the host server using SSL passthrough, also known as stream mode within this WAF platform.

However, nothing worked after creating the instance, and I noticed that port 443 is not being listened to when I executed the netstat command. So, I decided to restart the application, and I encountered the following error message.

![image](https://github.com/bunkerity/bunkerweb/assets/57866808/95b30337-3371-4876-96d5-da96f0bf09eb)

### How to reproduce?

Not sure how to reproduce this case, but I am certain that I selected the option to 'listen stream' and changed the non-SSL stream port to 80. I also changed the SSL stream port to 443. Reverse proxy is enabled as well.

Now, I am unable to access the web UI.

### Configuration file(s) (yaml or .env)

```YAML
# remove IS_LOADING=yes when your config is ready
#IS_LOADING=yes
HTTP_PORT=80
HTTPS_PORT=443
DNS_RESOLVERS=
API_LISTEN_IP=127.0.0.1
SERVER_NAME=xxxxxx.com
MULTISITE=yes
xxxxxx.com_USE_UI=yes
xxxxxx.com_USE_REVERSE_PROXY=yes
xxxxxx.com_REVERSE_PROXY_URL=/ui
xxxxxx.com_REVERSE_PROXY_HOST=http://127.0.0.1:7000
xxxxxx.com_INTERCEPTED_ERROR_CODES=400 404 405 413 429 500 501 502 503 504
```

### BunkerWeb version

1.5.5

### What integration are you using?

Linux

### Linux distribution (if applicable)

Ubuntu Server 22.04.2 LTS

### Removed private data

- [X] I have removed all private data from the configuration file and the logs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by TheophileDiot** [comment]
Hi @snow6326, thank you for opening this issue. Indeed the Templator wasn't rendering some common configuration files. This should be fixed with the last commit on the `dev` branch.

**Comment by lugg1** [comment]
Is a workaround possbile? Thanks.

**Comment by TheophileDiot** [comment]
> Is a workaround possbile? Thanks.

Hi @lugg1, you can either patch the file that has the issue like @snow6326 did. You can also use the `staging` tag or wait a bit longer until the next release that should be out in a couple weeks. 😁

**Comment by fl0ppy-d1sk** [comment]
Hello @lugg1 @snow6326,

Fix is now available in the v1.5.6 release. Thanks for your feedback !
